dap-etl/extraction/src/main/scala/org/broadinstitute/monster/dap/afus/AfusExtractionPipeline.scala

```diff
@@ -0,0 +1,98 @@
+package org.broadinstitute.monster.dap.afus
```

Comment:
I believe this was already in your other PR, is it possible this is a branch off of the other feature branch?